src/fidesctl/core/sync.py

```diff
@@ -42,10 +56,55 @@ def sync(manifests_dir: str, url: str, headers: Dict[str, str]) -> None:
                     updated_resource_list.append(resource)
 
             updated_manifest[resource_type] = updated_resource_list
-
-        with open(manifest_path, "w") as manifest_file:
-            yaml.dump(updated_manifest, manifest_file, sort_keys=False, indent=2)
-        echo_green(f"Updated manifest file written out to: '{manifest_path}'")
+            write_manifest_file(manifest_path, updated_manifest)
         print_divider()
 
+    return existing_keys
+
+
+def sync_missing_resources(
+    manifest_path: str, url: str, headers: Dict[str, str], existing_keys: List[str]
+) -> bool:
+    """
+    Writes all "system", "dataset" and "policy" resources out locally
+    that currently only exist on the server.
+    """
+
+    print(f"Writing out new resources to file: '{manifest_path}'...")
+    resources_to_sync = ["system", "dataset", "policies"]
+    resource_manifest = {
+        resource: list_server_resources(
+            url=url,
+            headers=headers,
+            resource_type=resource,
+            exclude_keys=existing_keys,
+            raw=True,
+        )
+        for resource in resources_to_sync
+    }
+
+    # Write out the resources in a file
+    write_manifest_file(manifest_path, resource_manifest)
+    print_divider()
+    return True
+
+
+def sync(
+    manifests_dir: str,
+    url: str,
+    headers: Dict[str, str],
+    sync_new: Optional[str],
+) -> None:
+    """
+    If a resource in a local file has a matching resource on the server,
+    write the server version into the local file.
+
+    If the 'all' flag is passed, additionally pull all other server resources
+    into local files as well.
+    """
+    existing_keys = sync_existing_resources(manifests_dir, url, headers)
+
+    if sync_new:
+        sync_missing_resources(sync_new, url, headers, existing_keys)
```

Comment:
@ThomasLaPiana  This feels more like it should be output-missing or export-missing

Reply:
`pull_missing_resources`? I think `pull` is actually more accurate since it isn't a sync...nothing is getting pushed up. Data is only getting pulled down

Reply:
+1 on pull, I'd say we s/sync/pull/g entirely